### ISSUE TITLE
[BUG] Bruno crash -  "TypeError: r.replace is not a function"

### ISSUE THREAD

**Issue by fractalf** [usebruno/bruno#382]
I have a request that just crashes bruno..
It works fin in Insomnia.

I'm on EndeavourOS using this package
https://aur.archlinux.org/packages/bruno-bin
v0.19.0

![image](https://github.com/usebruno/bruno/assets/5707457/b3b9de28-77fe-4d97-ab3a-ff18d28f7c03)

This is from a POST request with the following body:

```json
{
  "municipality_id": 1,
  "geographies": [
    {
      "zone_id": 98,
      "polygon_id": 167,
      "name": "Foo Bar",
      "geojson": {
        "type": "Feature",
        "geometry": {
          "type": "Polygon",
          "coordinates": [
            [
              [
                5.3222751617432,
                60.387673824597
              ],
              [
                5.3224790096283,
                60.387846120554
              ],
              [
                5.3230154514313,
                60.387702982438
              ],
              [
                5.3218621015549,
                60.386629426517
              ],
              [
                5.3209716081619,
                60.387019091646
              ],
              [
                5.3210520744324,
                60.387098614569
              ],
              [
                5.320799946785,
                60.387191391068
              ],
              [
                5.3202420473099,
                60.386849442095
              ],
              [
                5.3199148178101,
                60.38690643384
              ],
              [
                5.3199228644371,
                60.387029694714
              ],
              [
                5.3197807073593,
                60.387045599309
              ],
              [
                5.3198128938675,
                60.387289468792
              ],
              [
                5.3205049037933,
                60.387928292151
              ],
              [
                5.3209555149078,
                60.387960100455
              ],
              [
                5.3208965063095,
                60.38788588103
              ],
              [
                5.3221088647842,
                60.387567795862
              ],
              [
                5.3222751617432,
                60.387673824597
              ]
            ]
          ]
        }
      }
    }
  ]
}
```

**Comment by leandrosa81** [comment]
having the same issue, it started on v0.17.0. just to add some more detail, it seems the variable r is not a string with the response, but the already de-serialized JSON object:
![image](https://github.com/usebruno/bruno/assets/143780/73ac5536-0e30-4f7e-8e13-0575ba872e7f)

**Comment by helloanoop** [comment]
@fractalf @leandrosa81  We are now at `v0.22.0`

It's highly likely that this is already fixed. Can you install the latest version and check ?
There are also many many features that have been launched since `v0.17.0`

You can join our [community discord](https://discord.com/invite/KgcZUncpjq) to stay updated on the releases.

**Comment by helloanoop** [comment]
This helps @robert-wettstaedt 

I think I know the problem. Please standby.

**Comment by helloanoop** [comment]
I think this happens when the data is being sent as json, but with a `text/html ` as content type and the preview renderer gets activated and is causing the app to crash

cc @Its-treason

**Comment by robert-wettstaedt** [comment]
Yes I can confirm `v0.22.1` fixed it for me

**Comment by helloanoop** [comment]
This has been fixed in `v0.22.1`

**Comment by fractalf** [comment]
Great work guys! Keep it up :)
